docs/tutorial/part-one/index.md

```diff
@@ -81,18 +81,18 @@ Too cool 😎
 
 Gatsby's development server is a "hot reloading" server, meaning any change you
 make to your React.js page components (and later we'll learn, your data files)
-will hot reload in the browser.
+will be immediately visible and/or loaded in the browser.
```

Comment:
It's better to not use the term your explaining in your explanation.

Reply:
Haha yes :-)